options.go

```diff
@@ -531,7 +582,6 @@ func newConnPool(
 		PoolFIFO:        opt.PoolFIFO,
 		PoolSize:        opt.PoolSize,
 		PoolTimeout:     opt.PoolTimeout,
-		DialTimeout:     opt.DialTimeout,
```

Comment:
Is it an internal one or part of a public API?

Reply:
Actually, good catch, that was removed during the resolution of the merge conflicts. Should be put back